source/patterns/@aws-solutions-constructs/aws-lambda-kinesisstream/README.md

```diff
@@ -0,0 +1,115 @@
+# aws-lambda-kinesisstream module
+<!--BEGIN STABILITY BANNER-->
+
+---
+
+![Stability: Stable](https://img.shields.io/badge/cfn--resources-stable-success.svg?style=for-the-badge)
+
+---
+<!--END STABILITY BANNER-->
+
+| **Reference Documentation**:| <span style="font-weight: normal">https://docs.aws.amazon.com/solutions/latest/constructs/</span>|
+|:-------------|:-------------|
+<div style="height:8px"></div>
+
+| **Language**     | **Package**        |
+|:-------------|-----------------|
+|![Python Logo](https://docs.aws.amazon.com/cdk/api/latest/img/python32.png) Python|`aws_solutions_constructs.aws_lambda_kinesis_stream`|
+|![Typescript Logo](https://docs.aws.amazon.com/cdk/api/latest/img/typescript32.png) Typescript|`@aws-solutions-constructs/aws-lambda-kinesisstream`|
+|![Java Logo](https://docs.aws.amazon.com/cdk/api/latest/img/java32.png) Java|`software.amazon.awsconstructs.services.lambdakinesisstream`|
+
+## Overview
+This AWS Solutions Construct deploys an AWS Lambda Function that can put records on an Amazon Kinesis Data Stream.
+
+Here is a minimal deployable pattern definition:
+
+Typescript
+``` typescript
+import { Construct } from 'constructs';
+import { Stack, StackProps } from 'aws-cdk-lib';
+import { LambdaToKinesisStreamProps } from '@aws-solutions-constructs/aws-lambda-kinesisstream';
```

Comment:
OK, this is gonna be confusing. Our pattern so far is to name the file/construct after the service KinesisStreams. But Props and Properties are named after the resource - kinesisStreamProps. So your props/properties are OK, but the file/constructs need to be plural. Not sure we'd do it this way if we started from scratch today, but it's where we are now.

Reply:
I see - Fixed in latest commit.